.github/workflows/linux.yml

```diff
@@ -49,6 +49,13 @@ jobs:
         with:
           redis-version: 6
 
+      - name: run diesel migrations
+        uses: danitrod/diesel-cli-action@v0.3.4
+        with:
+          migration_directories: database_interactions/diesel
+          database_url: test.db
+          sqlite: true
```

Comment:
Looking at https://github.com/ducharmemp/diesel-cli-action, I don't think it's complicated enough to use an external action. I'd like to hear some thoughts from @robjtede.

Reply:
yes, please inline this action

Reply:
Alright, will try to inline it later today